### PR TITLE
Add SASS and break down sidebar into contact and education react components

### PR DESCRIPTION
**Pull request by DTThor**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,12 +15,17 @@ class App extends Component {
...
     return (
-      <div>
-        <SectionHeader name="CAREER PROFILE" icon="icon fas fa-user-circle fa-lg" />
-        <Profile />
-        <SectionHeader name="EXPERIENCE" icon="icon fas fa-briefcase fa-lg"/>
-        <ExperienceList />
-        <Sidebar />
+      <div className="container">
+        <div className="main">
+          <SectionHeader name="CAREER PROFILE" icon="icon fas fa-user-circle fa-lg" />
```
Comment (ashleymichal) [inline src/App.js]: semantically it seems the section headers belong in the components they are headers for.

File: src/components/Education.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+
+const Education = ({ title, degree, school, years }) => (
+ (
+
```
Comment (ashleymichal) [inline src/components/Education.js]: lots of extra whitespace in this file

**Review by ashleymichal — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (DTThor) [reply src/App.js]: thank you. done.

File: src/components/Education.js (hunk 2 above)
Comment (DTThor) [reply src/components/Education.js]: done.

File: package.json [hunk 3]
```diff
@@ -42,5 +41,9 @@
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-react": "^7.9.1",
     "eslint-plugin-standard": "^3.1.0",
+    "mini-css-extract-plugin": "^0.4.0",
```
Comment (lucas-aragno) [inline package.json]: can you remove this one? since we are not using it anymore

Comment (DTThor) [reply package.json]: done.

**Pull request merged**